api/app/serializers/spree/v2/storefront/store_serializer.rb

```diff
@@ -0,0 +1,13 @@
+module Spree
+  module V2
+    module Storefront
+      class StoreSerializer < BaseSerializer
+        set_type :store
+
+        attributes :name, :url, :meta_description, :meta_keywords, :seo_title, :mail_from_address, :default_currency, :code, :default,
+                   :supported_currencies, :facebook, :twitter, :instagram, :default_locale, :customer_support_email, :default_country_id, :description,
+                   :address, :contact_phone, :contact_email, :new_order_notifications_email
```

Comment:
@coolprobn I think we can skip `new_order_notifications_email` - that's usually an internal email address and should not be made public, as well as `mail_from_address` and `code`.

Besides that - great work! Thank you!

Reply:
Sure, I will make required changes and push again. Thank you for pointing them out.

Reply:
This has been resolved. Thank you!